helm/observability-bundle/values.yaml

```diff
@@ -78,9 +78,6 @@ apps:
     # It can be secret or configmap
     # https://github.com/giantswarm/rfc/tree/main/multi-layer-app-config#example
     extraConfigs:
```

Comment:
How will you set the remote write api key secret without a secret?

Reply:
They will be set via env vars from within another secret mount directly into Alloy pod. This secret is then not used by the app but by the pod directly.

Reply:
That looks weird. How is that secret created on thé WC?

Reply:
I found no other mechanism to pass secret values to Alloy, the only possible current configuration is a single configmap.
Good catch, this secret is not created on WC, I will need to change this to have the secret created by the app then (and port this to logging-operator too for Alloy logs)

Reply:
I think secret management is build in the grafana-agent app, either via env or something else. Maybe you Can check that?

Reply:
I added a Secret resource into our helm chart which allow to pass secret values via helm values.